04_removeFromArray/removeFromArray.js

```diff
@@ -1,6 +1,26 @@
-const removeFromArray = function() {
+//This line declares function that can accept multiple arguments and puts them in an array
+const removeFromArray = function (a, ...values) {
```

Comment:
Why not simply name it understandably from the beginning?
```suggestion
const removeFromArray = function (array, ...values) {
```

Reply:
Ummm something, something, file size.